actions/approve-pr/src/main.ts

```diff
@@ -3,12 +3,12 @@ import * as github from '@actions/github';
 import { getInput, getNumberInput } from '@dfinity/action-utils';
 
 import { approvePullRequest } from './approve-pull-request';
-import { enableAutoMerge } from './enable-auto-merge';
+import { enableAutoMerge, MergeMethod } from './enable-auto-merge';
 
 export async function run(): Promise<void> {
   try {
     const token = getInput('token');
-    const mergeMethod = getInput('merge_method');
```

Comment:
We were already asking for the input here, but it wasn't being supplied so this was throwing.